### ISSUE TITLE
Every Asset has a Content Type

### ISSUE THREAD

**Issue by georgemillard** [emfoundation/asset-manager#235]
Review and update Content Type field, make mandatory and assign to each Asset

[added by Alex]
"Final" list so far

Case study
Report
Co.Project
Paper
Infographic
Teaching & learning resources
Other

**Comment by asimonw** [comment]
@georgemillard can you confirm that the new list of content types is:

Case study
Report
Co.Project
Paper
Infographic
Lesson plan

Shall we add an "Other" type to capture assets which don't fit any of the above for now? I wonder how some internal assets, such as logos and other images will be captured (other than their format being Image or Graphic).

Content types which would need to be removed:

Image
Link
Presentation
Video
Workshop summary

I'm currently collecting a list of the affected assets.

**Comment by MertonL** [comment]
I'm capturing Colins suggestions from the Survey on Content Type here:
 

>I wondered how we’d list Ellen’s TED talk. None of the above suit, but **presentation** would. Equally, how would we categorise some of the DIF material, if it wasn’t simply a case study (i.e. Lewis Dartnell)? **Interview** would work.
> 
>Lesson plan might be a bit too ‘school’ for our university friends. Possible suggestions:
> 
>* ‘Learning activity’ is generic enough but sometimes not specific enough
>
>* ‘Educational resource’ again is generic enough, but we’re moving away from using the word ‘education’, so it might eventually be frowned upon.
>* ‘Teaching and learning resource’ covers the best of both options above, and works for HE and schools.
>* ‘HE resource’ - HE isn’t used in North America, but if there is a better term then it could be a useful content type for these resources, to be used alongside lesson plan
>
>Of all of the suggestions above, I prefer teaching and learning resource, even if it is a bit long.

**Comment by georgemillard** [comment]
Content Type List:

Case study
Report
Co.Project
Paper
Infographic
Lesson plan
Other

Following comments by Colin, do we need to rename Lesson Plan, (Teaching and Learning resource) or do we need to differentiate between Lesson Plans and Courses?

@asimonw Do you see content type as mutually exclusive? Could something be a case study and a report, for example? If so, I feel this field should not be required (unlike format). Perhaps then we should organise the assets into folders in the DAM by format.

**Comment by asimonw** [comment]
@georgemillard I think it's better to keep these mutually exclusive. For me a report and a case study are quite different things for example. I agree that it can get a bit tricky. I'd keep the groups universal enough. Maybe T&C resource is a better option to capture Lesson plans, courses etc. The word presentation is also a bit ambiguous, as it can mean something like a powerpoint (when it's a format) or for example a video of someone presenting (when it's a content type). 

Important note: if content types end up not being MECE (not required, or not one to one) then we cannot use them in the UI as currently suggested in the prototype. If that's the case we wouldn't be able to use them to group assets within a certain topic.

**Comment by georgemillard** [comment]
@asimonw good point. So if we have Content Type as required, but defaults to other, the field will enforce MECE-ness, so we can stick to the prototype plan and learn from that. I like Teaching and Learning Resource to replace Lesson Plan, and include courses etc.

**Comment by asimonw** [comment]
👍 I think that will keep things simpler for now indeed.

**Comment by asimonw** [comment]
@georgemillard This is still outstanding. Before we implement this, we need to agree on a strategy to handle existing assets which already have a content type that will be removed.

I've added the final list so far to the top comment for clarity.

**Comment by georgemillard** [comment]
@asimonw My thoughts would be to add the Other option first. Then programmatically set all those with deprecated Content Types to Other. Then remove deprecated options. Then update Other ones based on feedback from Nick (which I am currently confirming) to existing choices where possible.

**Comment by asimonw** [comment]
@georgemillard Solid plan!

**Comment by asimonw** [comment]
@georgemillard I'll do the first step, add the "Other" type now. I'll also revert the display text to title case (after a conversation with @fannyBkane we decided that that would be a better default for both DAM and cello).

One concern though: maybe it's better to make that change (just adding the "Other" option) as a hotfix to master, so that we can push that live before everything else. That way we can remove the unwanted content types from all the assets before removing those options from the database altogether.

**Comment by asimonw** [comment]
Created #257 to deal with the "Other" option first.

**Comment by asimonw** [comment]
Is now ready to be tested on staging

**Comment by MollyM42** [comment]
Assigning content type to first round DAM content I have spotted a possible gap with for articles/opinion pieces

**Comment by MollyM42** [comment]
@georgemillard 12 out of 70 so far. In can get a bit nuanced, for example some circulate articles I've classified as 'case studies' rather than 'other'

**Comment by asimonw** [comment]
@georgemillard All assets on the live site but 2 are now ready for the switch to the new content types. We still need to deploy the addition of the "Other" content type before we deploy the bigger changes. I'll add that to #264. 

One of the 2 assets which still need to be changed is new, a logo for the NPEC website. That raises the question: we don't have a content type to deal with internal website/design assets yet, right?

**Comment by fannyBkane** [comment]
@asimonw @georgemillard it was on the radar that logos don't have a content type. I had suggested to Molly to create a new type called 'comms content' (also including media packs, logos, brand guidelines...). Interestingly, what you're suggesting here is internal website assets, which is slightly different. 

What I would suggest, is to wait for more assets to be in that 'other' category to create a new type - wdyt?

**Comment by asimonw** [comment]
@fannyBkane that sounds like a good plan. Not all website content is comms content and vice versa, so probably needs a bit more thought. Looking at what we generate in practice first makes sense.

**Comment by georgemillard** [comment]
@asimonw @fannyBkane @MollyM42 I agree that waiting and seeing what falls into the other category, then creating new content types once a clear need is identified sounds like a good plan for now.

**Comment by dalwal08** [comment]
My take on the "Teaching and Learning Resources" type, a more useful name could be "Teaching and Learning Tools". This might mitigate the chance that case studies and infographics etc. get incorrectly classified. That way it is really only for lesson plans. curriculum guides and teaching material, but won't confuse or annoy school teachers or university lecturers.
